com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/expression/ExpressionGenerator.xtend

```diff
@@ -199,9 +199,17 @@ class ExpressionGenerator {
 		}
 	}
 	
+	private def StringConcatenationClient emptyToMapperJavaCode(RosettaExpression expr, ParamMap params, boolean multi) {
+		if (expr.isEmpty) {
+			'''«IF multi»«MapperC»«ELSE»«MapperS»«ENDIF».ofNull()'''
+		} else {
+			expr.javaCode(params)
+		}
+	}
+	
 	def StringConcatenationClient listLiteral(ListLiteral e, ParamMap params) {
 	    if (e.isEmpty) {
-	        '''«MapperS».ofNull()'''
+	        '''null'''
```

Comment:
This is actually the only change. The rest is already on master (not sure why it shows in the diff)